src/SprykerEco/Zed/Payone/PayoneConfig.php

```diff
@@ -70,7 +190,7 @@ class PayoneConfig extends AbstractBundleConfig
      *
      * @return string
      */
-    public function getMode()
+    public function getMode(): string
```

Comment:
It's a part of public API, please remove added type hints.

Reply:
Fixed.